apps/pmp-web/src/app/app.component.ts

```diff
@@ -1,7 +1,13 @@
 import { Component } from '@angular/core';
+// tslint:disable-next-line:nx-enforce-module-boundaries @ToDo how to handle such cases
+import { GoogleAnalyticsService } from '@pimp-my-pr/pmp-web/shared/util-google-analytics';
 
 @Component({
   selector: 'pmp-root',
   templateUrl: './app.component.html'
 })
-export class AppComponent {}
+export class AppComponent {
+  constructor(private googleAnalytics: GoogleAnalyticsService) {
+    this.googleAnalytics.init().then(_ => this.googleAnalytics.trackPageViews().subscribe());
```

Comment:
We don't want to trackPageViews in this app. You can leave the functionality but for now don't trigger it. You can also add an options to the init function with f. ex. init({trackPageViews: true}) for the future.